src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -886,4 +869,25 @@ occupied by other LAG group\(s\)./).matches(exc)
     def getLagCookie(portNumber) {
         new PortColourCookie(CookieType.LACP_REPLY_INPUT, portNumber).toString()
     }
+
+    private void assertSwitchHasCorrectLacpRulesAndMeters(SwitchExtended sw,
```

Comment:
It's better not to add assertion methods into specs. SwitchExtended can have methods to get expected and actual LACP rules and meteres.

Reply:
removed assertions to SwitchExtended class